### ISSUE TITLE
post request save data but shows empty response

### ISSUE THREAD

**Issue by prateekbansal63** [shabbirdwd53/Springboot-Microservice#9]
post request save data but shows empty response
<img width="960" alt="Capture111" src="https://user-images.githubusercontent.com/88434341/142729297-dbb63a0a-86e0-4b4c-88c0-46c41fd987ec.PNG">

**Comment by Divya-Rukmani** [comment]
It is because the data is not saved in the database.. Try adding it to the database.. 
After a lot of research, I got a solution in the spring-boot website [(https://spring.io/guides/tutorials/rest/)]
Add that DB which u have missed and launch the application.

**Comment by git-zaidism** [comment]
yeah you need to add this configuration class inside department service. And after that you can make post request via postman and it will work.


import org.slf4j.Logger;
import org.slf4j.LoggerFactory;
import org.springframework.boot.CommandLineRunner;
import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;

import com.zaidism.dept.entity.Department;
import com.zaidism.dept.repository.DeptRepository;

@Configuration
class LoadDatabase {

  private static final Logger log = LoggerFactory.getLogger(LoadDatabase.class);

  @Bean
  CommandLineRunner initDatabase(DeptRepository repository) {
    return args -> {
      log.info("Preloading " + repository.save(new Department()));
      log.info("Preloading " + repository.save(new Department()));
    };
  }
}

**Comment by Idimadakala** [comment]
Data is getting added to h2 database and id is populating and other fields are null. I assume it is because of Lombok data support in STS.
